components/youtube/new-videos.js

```diff
@@ -0,0 +1,65 @@
+const axios = require('axios')
+const youtube = {
+  type: "app",
+  app: "youtube_data_api",
+}
+
+
+module.exports = {
+  name: "Youtube - New Videos",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    youtube,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  methods: {
+    async getVideos(pageToken=null) {
+      return await axios.get('https://www.googleapis.com/youtube/v3/search', {
+        headers: {
+          'Authorization' : 'Bearer '+this.youtube.$auth.oauth_access_token
+        },
+        params: {
+          'part' : 'snippet',
+          'type' : 'video',
+          'forMine' : true,
+          'pageToken' : pageToken
+        }
```

Comment:
You can simplify this a bit by removing quotes around property names and using object shorthand notation (since `pageToken` holds the same property name and variable reference) to get:

```
params: {
  part: 'snippet',
  type: 'video',
  forMine: true,
  pageToken,
}
```

Reply:
Updated this.